### PR TITLE
Typescript (Fix localization and custom title bar)

### PR DESCRIPTION
**Pull request by KiranNiranjan**
## Description
Fix localization and custom title bar bugs

## Changes
Typescript - Fix localization for screen picker & spellchecker modules
Typescript - Fix custom title bar menu item
Typescript - Fix screen picker unit tests

## QA Checklist
- [X] Unit-Tests
- [ ] Automation-Tests

### CODE REVIEW

File: src/app/app-menu.ts [hunk 1]
```diff
@@ -251,6 +261,19 @@ export class AppMenu {
                     : i18n.t('Bring to Front on Notifications')(),
                 type: 'checkbox',
             },
+            isWindowsOS
+                ? {
```
Comment (VishwasShashidhar) [inline src/app/app-menu.ts]: let's put this in a variable, far easier to read.

File: src/app/app-menu.ts [hunk 2]
```diff
@@ -251,6 +261,19 @@ export class AppMenu {
...
+                    label: this.titleBarStyle === TitleBarStyles.NATIVE
+                        ? i18n.t('Enable Hamburger menu')()
+                        : i18n.t('Disable Hamburger menu')(),
+                    click: () => {
```
Comment (VishwasShashidhar) [inline src/app/app-menu.ts]: same as above

File: src/renderer/screen-snippet-bc-handler.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { ipcRenderer } from 'electron';
+import { apiCmds, apiName, IScreenSnippet } from '../common/api-interface';
+
+/**
+ * @deprecated user openScreenSnippet instead
+ */
+export class ScreenSnippetBcHandler {
```
Comment (VishwasShashidhar) [inline src/renderer/screen-snippet-bc-handler.ts]: what is `bc`?

Comment (KiranNiranjan) [reply src/renderer/screen-snippet-bc-handler.ts]: `Backward Compatibility`

**Pull request merged**